contracts/proposals/ListOnZoraProposal.sol

```diff
@@ -142,21 +134,14 @@ contract ListOnZoraProposal is ZoraHelpers {
         uint40 timeout,
         // How long the auction will run for once a bid has been placed.
         uint40 duration,
-        IERC721 token,
+        address token,
         uint256 tokenId
-    ) internal override returns (uint256 auctionId) {
-        token.approve(address(ZORA), tokenId);
-        auctionId = ZORA.createAuction(
-            tokenId,
-            token,
-            duration,
-            listPrice,
-            payable(address(0)),
-            0,
-            IERC20(address(0)) // Indicates ETH sale
-        );
+    ) internal virtual override {
+        IERC721(token).approve(address(ZORA.erc721TransferHelper()), tokenId);
```

Comment:
Since `erc721TransferHelper` is an immutable in `ReserveAuctionCoreErc20`, we can store this as an immutable in the contract and save an external call.